src/virtualship/instruments/xbt.py

```diff
@@ -0,0 +1,125 @@
+"""XBT instrument."""
+
+from dataclasses import dataclass
+from datetime import timedelta
+from pathlib import Path
+
+import numpy as np
+from parcels import FieldSet, JITParticle, ParticleSet, Variable
+
+from ..spacetime import Spacetime
+
+
+@dataclass
+class XBT:
+    """Configuration for a single XBT."""
+
+    spacetime: Spacetime
+    min_depth: float
+    max_depth: float
+
+
+_XBTParticle = JITParticle.add_variables(
+    [
+        Variable("temperature", dtype=np.float32, initial=np.nan),
+        Variable("max_depth", dtype=np.float32),
+        Variable("min_depth", dtype=np.float32),
+        Variable("winch_speed", dtype=np.float32),
+    ]
+)
+
+
+def _sample_temperature(particle, fieldset, time):
+    particle.temperature = fieldset.T[time, particle.depth, particle.lat, particle.lon]
+
+
+def _xbt_cast(particle, fieldset, time):
+    # lowering
+    particle_ddepth = -particle.winch_speed * particle.dt
+    if particle.depth + particle_ddepth < particle.max_depth:
+        particle.raising = 1
```

Comment:
Why set the raising flag here? An XBT doesn't get raised? Should the particle not simply be deleted here?